packages/@azure/servicebus/data-plane/README.md

```diff
@@ -1,22 +1,30 @@
-# @azure/service-bus
+# Azure Service Bus client library for Typescript and Javascript
```

Comment:
JavaScript & TypeScript

* Ampersands are great in headers.
* We could maybe get away with just JavaScript, though. If we don't specify Node and Browser, perhaps people will assume it works anywhere? Might be worth studying. https://twitter.com/bterlson/status/1113142840203239425

Reply:
How about 

> Azure Service Bus client library for Javascript

And once we have browser support, we can add a line below that says (not in H1), 

> This library can be used both in Nodejs and Browser applications

Reply:
Updated to

> Azure Service Bus client library for Javascript

And updated `Use the client library for Azure Service Bus to` to `Use the client library for Azure Service Bus in your Nodejs application to`

Once we have browser support we can change to

`Use the client library for Azure Service Bus in your Nodejs or Web application to`